data/transactions/logic/TEAL_opcodes.md

```diff
@@ -18,7 +18,7 @@ Ops have a 'cost' of 1 unless otherwise specified.
 - SHA256 hash of value X, yields [32]byte
 - **Cost**:
    - 7 (LogicSigVersion = 1)
-   - 35 (2 <= LogicSigVersion <= 5)
+   - 35 (LogicSigVersion => 2)
```

Comment:
`>=` would be more conventional, and less confused with an arrow?

Reply:
oh, geez, you're right of course.  thanks.